cpp/doxygen/developer_guide/DOCUMENTATION.md

```diff
@@ -198,45 +192,39 @@ Also, try to include a short [example](#inline-examples) if possible.
 
 ### @brief
 
-The `@brief` text should be a short, one sentence description.
+The [\@brief](http://www.doxygen.nl/manual/commands.html#cmdbrief) text should be a short, one sentence description.
 Doxygen does not provide much space to show this text in the output pages.
-Always follow the `@brief` line with a blank comment line.
+Always follow the \@brief line with a blank comment line.
 
 The longer description is the rest of the comment text that is not tagged with any doxygen command.
 
-```c++
-/**
- * @brief Short description.
- *
- * Long description.
- *
-```
+    /**
+     * @brief Short description.
+     *
+     * Long description.
+     *
 
-### @copydoc
+### \@copydoc
 
 Documentation for declarations in headers should be clear and complete.
-You can use the `@copydoc` tag to avoid duplicating the comment block for a function definition.
-
-```c++
-  /**
-   * @copydoc complicated_function(int,double*,float*)
-   *
-   * Any extra documentation.
-   */
-```
-
-Also, `@copydoc` is useful when documenting a `detail` function that differs only by the `stream` parameter.
-
-```c++
-/**
- * @copydoc cudf::segmented_count_set_bits(bitmask_type const*,std::vector<size_type> const&)
- *
- * @param[in] stream Optional CUDA stream on which to execute kernels
- */
-std::vector<size_type> segmented_count_set_bits(bitmask_type const* bitmask,
-                                                std::vector<size_type> const& indices,
-                                                rmm::cuda_stream_view stream = cudf::default_stream_value);
-```
+You can use the [\@copydoc](http://www.doxygen.nl/manual/commands.html#cmdcopydoc) tag to avoid duplicating the comment block for a function definition.
```

Comment:
```suggestion
You can use the [\@copydoc](https://www.doxygen.nl/manual/commands.html#cmdcopydoc) tag to avoid duplicating the comment block for a function definition.
```